src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx

```diff
@@ -239,6 +278,17 @@ export function RepeatingGroupTable({
     }
   };
 
+  const onClickSave = () => {
```

Comment:
I would prefer to name this `handleSaveClick` instead, as its handling the event. That makes it easy to spot where its used, if the event is just passed further down, for some other component to handle.

Related blog post https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

```suggestion
  const handleSaveClick = () => {
```